request.go

```diff
@@ -1926,6 +1926,62 @@ func (r *Request) WithMultipart() *Request {
 	return r
 }
 
+//	req := NewRequestC(config, "GET", "/path")
+//	req.WithClient(&http.Client{
+//	  Transport: &http.Transport{
+//		DisableCompression: true,
+//	  },
+//	})
+
+// WithReporter sets reporter.
+//
+// The new reporter overwrites AssertionHandler.Reporter. It will be used to
+// report formatted fatal failure messages
+//
+// Example:
+//
+//	req := NewRequestC(config, "GET", "http://example.com/path")
+//	reporter := httpextect.ReporterFunc(
+//		func(message string, args ...interface{}) {
+//			// reporter code here
+//		}),
+//	req.WithReporter(reporter)
+func (r *Request) WithReporter(reporter Reporter) *Request {
+	return r.WithAssertionHandler(&DefaultAssertionHandler{
+		Reporter:  reporter,
+		Formatter: r.config.Formatter,
+	})
+}
+
+// WithAssertionHandler sets assertion handler.
+//
+// The new handler overwrites assertion handler. It will be used to
+// format and report test Failure or Success.
+//
+// Example:
+//
+//	req := NewRequestC(config, "GET", "http://example.com/path")
+//	req.WithAssertionHandler(DefaultAssertionHandler)
```

Comment:
Could you please use a correct argument in example, instead of "DefaultAssertionHandler".

I mean, something like:

```go
&DefaultAssertionHandler{
   <initializers here>
```